MEME/MEME/Model/MyPage/ModelMyPageViewController.swift

```diff
@@ -59,44 +61,35 @@ class ModelMyPageViewController: UIViewController, ModelHeaderViewDelegate {
                 
                 tableView.layoutIfNeeded()
             }
-
-//        getMyPageProfile (userId: 1)
-//        self.getMyPageProfile(userId: KeyChainManager.loadMemberID())
-        
-     
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(true)
         self.tabBarController?.tabBar.isHidden = false
     }
-    // MARK: - Helpers
     
-    func configureUI() {
-        
-        tableView.backgroundColor = .white
-        
+    // MARK: - Helpers
+    func setupViews() {
         view.addSubview(tableView)
-        tableView.translatesAutoresizingMaskIntoConstraints = false
         
         tableView.dataSource = self
         tableView.delegate = self
         tableView.separatorStyle = .none
-        
         tableView.register(ModelMyPageTableViewCell.self, forCellReuseIdentifier: cellID)
```

Comment:
```suggestion
        tableView.register(ModelMyPageTableViewCell.self, forCellReuseIdentifier: ModelMyPageTableViewCell.className)
```

과 같이 수정하면 더 좋을 것 같습니다.